### ISSUE TITLE
Extend metadata.json

### ISSUE THREAD

**Issue by bastelfreak** [dhoppe/puppet-resolv#1]
Hey,

I tested this module on CentOS 6 and 7, Archlinux, Gentoo and FreeBSD 10.1 and didn't notice any issues. You can extend the metadata.json if you want. Let me know if you want me to run specific tests on these distributions. My code:

``` puppet
  class {'::resolv':
    config_file_template  => 'resolv/common/etc/resolv.conf.erb',
    nameservers           => ['2a01:4f8:0:a0a1::add:1010', '2a01:4f8:0:a102::add:9999', '2a01:4f8:0:a111::add:9898', '213.133.98.98', '213.133.99.99', '213.133.100.100',],
  }
```

feature request: make resolv/common/etc/resolv.conf.erb' the default value of config_file_template instead of undef?

**Comment by dhoppe** [comment]
@bastelfreak Thank you very much. I would appreciate a PR against the develop branch. ;)

**Comment by dhoppe** [comment]
This issue has been solved via #3.
